components/infobox/extensions/wikis/stormgate/infobox_extension_attack.lua

```diff
@@ -47,14 +47,26 @@ function Attack.run(argsJson, attackIndex, faction)
 
 	return {
 		Title{name = 'Attack' .. attackIndex .. ': ' .. args.name},
-		Cell{name = 'Target', content = data.targets},
-		Cell{name = 'Damage', content = {data.damagePercentage and (data.damagePercentage .. '%') or data.damage}},
-		Cell{name = 'Effect', content = Array.map(data.effect, function(effect) return Page.makeInternalLink(effect) end)},
+		Cell{name = 'Target', content = {Attack._displayCommaSeparatedString(args.target)}},
```

Comment:
Because I couldn't change it in Attack._parse directly since the data is also used for storage and when I changed it there, I destroyed the storage. 

I though it would be easier for the Cells, which show comma seperated links, to directly parse them from the args-strings since I can use the same function everytime this way. This regards Target, Effect and Bonus. 

(In data there is the array for targets and effects and a string for bonus. )

Reply:
done